cobra/test/__init__.py

```diff
@@ -50,7 +53,7 @@ def test_all(args=None):
 
         return pytest.main(
             ['--pyargs', 'cobra', '--benchmark-skip', '-v', '-rs'] + args
-        ) != 0
+        )
```

Comment:
what is now the return value? do the [install instructions](https://github.com/opencobra/cobrapy/blob/devel/INSTALL.rst#testing-your-installation) need to be updated?